environment/commands/init-near2eth-prover.js

```diff
@@ -0,0 +1,43 @@
+const Web3 = require('web3');
+const fs = require('fs');
+const { RainbowConfig } = require('../lib/config');
+
+class InitNear2EthProver {
+    static async execute () {
+        // @ts-ignore
+        const web3 = new Web3(RainbowConfig.getParam('eth-node-url'));
+        let ethMasterAccount =
+            web3.eth.accounts.privateKeyToAccount(RainbowConfig.getParam('eth-master-sk'));
+        web3.eth.accounts.wallet.add(ethMasterAccount);
+        web3.eth.defaultAccount = ethMasterAccount.address;
+        ethMasterAccount = ethMasterAccount.address;
+
+        // Initialize client contract.
+        console.log('Deploying Near2EthClient contract.');
+        const nearProver = new web3.eth.Contract(JSON.parse(
+            // @ts-ignore
+            fs.readFileSync(RainbowConfig.getParam('near2eth-prover-abi-path'))));
```

Comment:
can also use unblocking version:
```
const fs = require('fs').promises;
await fs.readFile(RainbowConfig.getParam('near2eth-prover-abi-path'))));
```
but i guess readFileSync it's fine

Reply:
Yes, we are using sync version everywhere in this code..